lib/router.js

```diff
@@ -45,18 +46,26 @@ router.get('/slashCommands/ci', (req, res) => {
 router.get('/slashCommands/ext', (req, res) => {
     const command = req.query && req.query.command;
     const text = req.query.text;
+    const update = req.query.update;
```

Comment:
Your line of thought here is generally correct, but we have to remember that the end user (a Slack user) won't be able to add a query param to the request URL... A slack user can only type something like: `/<comand> <arbitrary text>`, right? That's actually all we have in the request: `req.query.command` and `req.query.text`. Hence, my suggestion is to check `req.query.text` and see if it's `=== 'update'` and if it's, then we trigger the update process... like so: `/ext update`. What do you think of this approach?